src/components/SimplePagination/_simple-pagination.scss

```diff
@@ -0,0 +1,28 @@
+@import '~carbon-components/scss/globals/scss/vars';
+
+div.simple-pagination-container {
```

Comment:
We should use our new iot prefix for our styles moving forward.

Reply:
Fixed